src/cache_invalidate.c

```diff
@@ -101,6 +102,21 @@ cache_invalidate_callback(Datum arg, Oid relid)
 	}
 }
 
+static void
+cache_invalidate_callback(Datum arg, Oid relid)
+{
+	/* original callback */
+	cache_invalidate_callback_internal(arg, relid);
+
+	/* invalidate cache in parallel worker */
+	if (IsParallelWorker())
```

Comment:
No need to call `RelationCacheInvalidate` again. It will happen via the PG backend invalidation processing automatically.